package.json

```diff
@@ -33,10 +33,13 @@
     "esbuild": "^0.15.5",
     "firebase": "^9.9.3",
     "firebase-admin": "^11.5.0",
+    "fs": "^0.0.1-security",
     "intro.js": "^3.3.1",
     "jspdf": "^2.5.1",
     "jspdf-autotable": "^3.5.25",
     "node-fetch": "^2.6.7",
+    "pdf-to-excel": "^0.3.2",
+    "pdfjs-dist": "^3.4.120",
```

Comment:
Can we change these to developer dependencies so they won't be bundled with the app sent to the client? You can first uninstall them, and then reinstall them with `npm install --save-dev` so that they are installed as dev dependencies.